uni/lib/utils/navbar_items.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+enum NavbarItem {
+  navPersonalArea(
+    Icons.home_outlined,
+    ['area', ''],
+  ),
+  navAcademicPath(
+    Icons.school_outlined,
+    ['horario', 'exames', 'cadeiras'],
+  ),
+  navRestaurants(
+    Icons.local_cafe_outlined,
+    ['restaurantes'],
+  ),
+  navFaculty(
+    Icons.domain_outlined,
+    ['biblioteca', 'uteis', 'calendario'],
+  ),
+  navTransports(Icons.map_outlined, ['locais', 'autocarros']);
+
+  const NavbarItem(this.icon, this.routes);
+
+  final IconData icon;
+  final List<String> routes;
+
+  // TODO(Process-ing): Transform into single route when the new pages are done
```

Comment:
Let's wait for #1068 and #1067 before merging this, as this now renders some pages useless.